### ISSUE TITLE
Add support for Host Facts

### ISSUE THREAD

**Issue by bdonnahue** [theforeman/foreman-ansible-modules#213]
##### SUMMARY
Foreman exposes an API to add Facts to a Host:
POST /api/hosts/facts

I would like to be able to encapsulate this in a playbook
Would it be possible to add a module to support this functionality?

##### ISSUE TYPE
 - Feature Idea

**Comment by mdellweg** [comment]
Reading the facts might be possible already with the help of the `foreman_search_facts` module.
(That is if there is a corresponding Nailgun entity.)
Setting them is, i think, against the design of host facts.
@sean797 any idea?

**Comment by sean797** [comment]
I don't think Adding Facts to a Host shouldn't be done manually, I'd consider using `Global parameters`. If you want Ansible upload facts to Foreman, I think the foreman_ansible plugin can do that.

**Comment by bdonnahue** [comment]
@sean797 thanks for reccomending the foreman_ansible plugin. I think this is what I need. I will check it out and report back.

**Comment by bdonnahue** [comment]
Yes I think that will do it. Please feel free to close this issue and point to that plugin
